2009-04-09  Michael Tautschnig  <mt@debian.org>

	* fai-mirror: Set Dir::Etc option to $aptcache/etc/apt instead of changing
		Dir::Etc::sourceslist only (thanks Sebastian Harl) (closes: #504813)
Index: trunk/bin/fai-mirror
===================================================================
--- trunk.orig/bin/fai-mirror
+++ trunk/bin/fai-mirror	
@@ -136,7 +136,7 @@
       -o Dir::Cache=$aptcache/var/cache/apt \
       -o Dir::State=$aptcache/var/cache/apt \
       -o Dir::Cache::Archives=$archivedir \
-      -o Dir::Etc::sourcelist=$aptcache/etc/apt/sources.list \
+      -o Dir::Etc=$aptcache/etc/apt/ \
       -o Dir::State::Lists=$aptcache/var/lib/apt/lists/"
 
     # not needed
